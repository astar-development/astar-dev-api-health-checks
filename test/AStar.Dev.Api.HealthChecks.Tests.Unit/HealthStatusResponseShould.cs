using AStar.Dev.Utilities;
using JetBrains.Annotations;

namespace AStar.Dev.Api.HealthChecks;

[TestSubject(typeof(HealthStatusResponse))]
public class HealthStatusResponseShould
{
    [Fact]
    public void ContainTheExpectedProperties()
    {
        new HealthStatusResponse
            {
                Name                   = "Test Name",
                Description            = "Test Description",
                DurationInMilliseconds = 123,
                Data                   = new Dictionary<string, object>(),
                Exception              = "Test Exception",
                Status                 = "OK"
            }
            .ToJson()
            .ShouldMatchApproved();
    }
}
