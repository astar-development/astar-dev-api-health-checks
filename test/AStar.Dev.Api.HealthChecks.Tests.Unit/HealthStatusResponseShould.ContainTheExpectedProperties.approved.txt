{
  "name": "Test Name",
  "description": "Test Description",
  "status": "OK",
  "durationInMilliseconds": 123,
  "data": {},
  "exception": "Test Exception"
}